dashboard/test/ui/features/funOMeter.feature

```diff
@@ -2,13 +2,12 @@
 Feature: Fun-O-Meter
 
 Scenario: Rate a Puzzle
-  Given I am on "http://learn.code.org/s/allthethings/stage/2/puzzle/3?noautoplay=true"
+  Given I am on "http://learn.code.org/s/allthethings/stage/4/puzzle/4?noautoplay=true"
   And I rotate to landscape
   And I wait to see "#x-close"
   And I close the dialog
 
-  When I drag block "1" to block "4"
-  And I drag block "1" to block "5"
+  When I drag block "4" to block "11" plus offset 35, 30
```

Comment:
Not a big deal, but if you want to help with test readability you can alias blocks instead of using numbers, [as done here in blockTrashing.feature](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/ui/features/blockTrashing.feature#L12).